<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox::indicator:unchecked{
	image: url(:/icon/QCheckBox-unchecked.png);
}

QCheckBox::indicator:checked{
	image: url(:/icon/QCheckBox-checked.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="step01">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step02">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step03">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step04">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step05">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step06">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step07">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step08">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>61</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>61</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="errorView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="step17">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step16">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step11">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step18">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step13">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step14">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step27">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step22">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step26">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step21">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step25">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step28">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step23">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step24">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>kick</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>hat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>clap</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="tempoBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="step09">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step0A">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step0C">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step0B">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step0E">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step0D">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step0G">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step0F">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step1A">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step1E">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step1F">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step19">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step1B">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step1C">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step1D">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step1G">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step2A">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step2E">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step2F">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step29">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step2B">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step2C">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step2D">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="step2G">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
